indico/modules/events/requests/notifications.py

```diff
@@ -66,15 +66,17 @@ def notify_new_modified_request(req, new):
 
 
 @email_sender
-def notify_withdrawn_request(req, email_event_managers):
+def notify_withdrawn_request(req, email_event_managers, email_request_managers):
```

Comment:
We always pass those as kwargs anyway, so it's a good opportunity to make them kw-only

```suggestion
def notify_withdrawn_request(req, *, email_event_managers, email_request_managers):
```

Reply:
@ThiefMaster thanks for the review, I have made all the required changes